imagepipeline-base/src/main/java/com/facebook/cache/disk/DynamicDefaultDiskStorage.java

```diff
@@ -71,6 +71,15 @@ public boolean isEnabled() {
       return false;
     }
   }
+ 
+ @Override
+  public boolean isExternal() {
+    try {
+      return get().isExternal();
```

Comment:
I don't think `isExternal` can throw `IOException` as it's already handled there.